tests/src/Simple/ComWrappers/ComWrappersNative.cpp

```diff
@@ -0,0 +1,383 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#include <xplatform.h>
+#include <ComHelpers.h>
+#include <unordered_map>
+#include <list>
+#include <inspectable.h>
+
+namespace API
+{
+    // Documentation found at https://docs.microsoft.com/windows/win32/api/windows.ui.xaml.hosting.referencetracker/
+    class DECLSPEC_UUID("64bd43f8-bfee-4ec4-b7eb-2935158dae21") IReferenceTrackerTarget : public IUnknown
```

Comment:
I do not think you need to worry about the reference tracking. It is for WinRT only - not required for what you are trying to achieve.